src/components/menu/menu.scss

```diff
@@ -38,6 +36,12 @@ ion-menu.show-menu {
   transform: translate3d(-9999px, 0, 0);
 
   contain: strict;
+
+  // scss-lint:disable PropertySpelling
+  @include multi-dir() {
+    left: 0;
+    right: auto;
+  }
```

Comment:
Leave `menu.scss` unchanged
The correct default is `start`, and thus `0, auto, 0, 0` is correct
You are correct that there is a need to address left, so https://github.com/ionic-team/ionic/pull/11953

Sorry for all of the mess around this.